scene/resources/skeleton_profile.cpp

```diff
@@ -432,6 +432,9 @@ void SkeletonProfile::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_group", "bone_idx"), &SkeletonProfile::get_group);
 	ClassDB::bind_method(D_METHOD("set_group", "bone_idx", "group"), &SkeletonProfile::set_group);
 
+	ClassDB::bind_method(D_METHOD("is_require", "bone_idx"), &SkeletonProfile::is_require);
+	ClassDB::bind_method(D_METHOD("set_require", "bone_idx", "require"), &SkeletonProfile::set_require);
```

Comment:
Should we follow the naming if the c++ methods here or use `is_required`?

Reply:
It should be `set`/`is_required`.
I would also rename the C++ methods and struct member, if we don't mind a minor compat breakage for C++ modules.

Reply:
Will do!